wd_helper.au3

```diff
@@ -1389,7 +1389,7 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 				If $sDriverLatest > $sDriverVersion Then
 					$bResult = True
 				EndIf
-			ElseIf $sDriverLatest > $sDriverVersion Or $bForce Then
+			ElseIf $sDriverLatest > $sDriverVersion Or $bForce Or $sDriverVersion = 'None' Then
```

Comment:
I'd have to go back and check past discussions, but I seem to recall that I previously told others that they needed to set $bForce = True in the case where the webdriver didn't previously exist.

Reply:
Found it. See discussion [here](https://www.autoitscript.com/forum/topic/201106-webdriver-udf-help-support-ii/?do=findComment&comment=1467771)

Reply:
> so why `$bForce .... Default is False` ?
> Is this for security reason that Coders/Scripter must intentionaly download file ?

No. It's simply the way the function was designed to work.

> And the description is not accurate I mean misleading at least I was not aware about how to use them.

Can you be more specific? What would you change? We can certainly add more to the Remarks that clarify the functionality.

Reply:
> I do not want to update driver each run .
So how do you upgrade, but only if a newer version is available or no driver is available?

I see your point. In the current situation, you would need to call _WD_GetWebDriverVersion first, and then adjust your call to _WD_UpdateDriver as needed if the driver isn't present.

Reply:
> > And the description is not accurate I mean misleading at least I was not aware about how to use them.
> 
> Can you be more specific? What would you change? We can certainly add more to the Remarks that clarify the functionality.

Current description is:
`$bForce      - [optional] Force update? Default is False`

but it act like:
`$bForce      - [optional] Force download? Default is False`

because if you set True then it always download if you current local version is correct, which mean that you do not update but re download, which mean you forced to download regardles of any case. For me update mean only if there is lack of current version.

Reply:
> Force update? vs Force download?

To me, they pretty much mean the same thing because you have to download the update in order to apply it, correct? In this scenario, how would you perform an update without downloading the file?

Also, the function name is _WD_UpdateDriver, not _WD_DownloadDriver. Thus the existing description works for me. <shrug>

Reply:
Force update mean only when there is no dirver available or the driver is not up to date.
Force download .... regardless of the version you have - that is always.

Reply:
Current version of this function works exactly in this way.